### ISSUE TITLE
Please help configure Google Analytics with react-native-firebase

### ISSUE THREAD

**Issue by denieler** [invertase/react-native-firebase#1486]
I've configured `react-native-firebase` properly and it works for me. But I'm trying to analyze data using Google Analytics and I don't see any events in my GA account.

I have set up `google-services.json`:

```
....
      "services": {
        "analytics_service": {
          "status": 2,
          "analytics_property": {
            "tracking_id": "UA-*******"
          }
      },
...
```

I couldn't find any documentation connected to that.
Thank you so much for your help!

**Comment by denieler** [comment]
@Ehesp hey! Thanks for the answer, but I did everything that was in that guide you've shared and nothing happened. I saw all the data in Firebase Analytics, but zero information in Google Analytics. Are there any secrets I've missed?

**Comment by Ehesp** [comment]
Hmm that's strange. How long have you given it? 

There's not really anything else we can do, as once it goes off to Firebase it's out of our hands. I'd suggest opening a support ticket if it's been more than 48 hours of no data.

**Comment by denieler** [comment]
No, it's already happening during few weeks. I've tried to check many different resources already to solve this issue, but no success. Is it enough to put `tracking_id` configuration as I described to allow GA to receive tracking data?

**Comment by AshiHarel** [comment]
Hi @denieler, 
did you mannage to solve this issue? 
If so, how?

**Comment by denieler** [comment]
@AshiHarel no, just started to use `Firebase Analytics`, they released a better UI recently, so using it more or less acceptable for my needs
